lib/livebook/hubs/enterprise_client.ex

```diff
@@ -97,4 +114,22 @@ defmodule Livebook.Hubs.EnterpriseClient do
   defp broadcast_message(message) do
     Phoenix.PubSub.broadcast(Livebook.PubSub, @pubsub_topic, message)
   end
+
+  defp registry_name(%Enterprise{url: url}) do
+    {:via, Registry, {@registry, url}}
```

Comment:
Why don't we use the id to make sure we have one connection per enterprise instance?

Reply:
Because if you have 2 Enterprise, in both, you could have the same ID. We can't create two `enterprise-123` Hubs and we can't connect to both

Reply:
In this use case, what your thoughts?

Reply:
Ah, perhaps we should adjust the id to make sure it's unique? Non-unique ids are likely to cause issues. Should we hash the hostname and include as part of the id, any other ideas?

Reply:
We're going to use an unique ID.